google/ggrc-core
# GGRC-8846 Replace can-map with can-define-map for Reminder and Unarchive-link component

# Issue description

Replace usage of legacy package

# Steps to test the changes

1. Open any archived Audit
2. Click on 3bbs and click unarchive
3. Confirm action
4. Expected: It works
5. Open assessment with In Progress/Not Started state
6. Click on 3bbs menu and click reminder assignee
7. Confirm action
8. Expected: It works

# Solution description

Replace can-map with can-define-map for Reminder component and Unarchive link component
 
# Sanity checklist




# PR Review checklist
